requirements.txt

```diff
@@ -1,4 +1,4 @@
-psycopg2==2.6.2
+psycopg2==2.7.1
```

Comment:
You recently merged this change to master: you should not have to duplicate it here! Try to keep your local master up-to-date with the remote master, especially before checking out a new branch. Let me know if you have questions about this.

Reply:
👌🏻